qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/create/offer/ProjectManagerSelectionView.groovy

```diff
@@ -33,9 +36,10 @@ class ProjectManagerSelectionView extends VerticalLayout{
     ProjectManagerSelectionView(CreateOfferViewModel viewModel){
         this.viewModel = viewModel
         initLayout()
-        setupDataProvider()
+        def projectManagerDataProvider = setupDataProvider()
         generateCustomerGrid()
         addListener()
+        setupFilters(projectManagerDataProvider)
```

Comment:
Here I also would suggest moving the grid generation-specific methods into the grid generation method.